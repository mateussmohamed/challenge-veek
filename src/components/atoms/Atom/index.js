import styled from 'styled-components'

const Atom = styled.div`
  background-color: green;
`

export default Atom
