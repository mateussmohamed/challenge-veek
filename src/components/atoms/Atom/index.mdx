---
name: Atom
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import Atom from './index'

# Atom
<PropsTable of={Atom} />

<Playground>
    <Atom>Default</Atom>
</Playground>
