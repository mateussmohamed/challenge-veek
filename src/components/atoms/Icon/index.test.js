import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import Atom from './index'

describe('[<Atom />]', () => {
  const wrap = (props = {}) => shallow(<Atom {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({ name: 'facebook' })

    expect(wrapper).toMatchSnapshot()
  })
})
