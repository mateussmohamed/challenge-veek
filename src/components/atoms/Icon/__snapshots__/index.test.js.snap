// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<Atom />] matches to snapshot 1`] = `
<FacebookIcon
  name="facebook"
/>
`;
